llvm/utils/TableGen/RegisterBankEmitter.cpp

```diff
@@ -289,6 +295,81 @@ void RegisterBankEmitter::emitBaseClassImplementation(
      << "} // end namespace llvm\n";
 }
 
+// RegisterBankInfo tables and enums
+// are discussed in https://discourse.llvm.org/t/74459
```

Comment:
There are 21 posts in that thread at the time of writing. Can you remove the link and just put a summary of what this does?

Reply:
+1 on this

Reply:
Done.